lms/djangoapps/lti_provider/tasks.py

```diff
@@ -105,10 +105,13 @@ def send_composite_outcome(user_id, course_id, assignment_id, version):
             assignment.id
         )
         return
+    print '**********course_id', course_id
```

Comment:
Syntax Trivia: In Python, the multiplication operator `*` is overloaded for strings such that a string multiplied by a number returns a string with said character repeated `N` times.

Try this:
```sh
python -c "print('*' * 10)"
```

prints

> '**********'